apps/frontend/src/app/pages/company/profile/profile.component.ts

```diff
@@ -54,27 +111,119 @@ export class ProfileComponent {
       this.background = fileUrl;
     } else {
       this.profilePhotoFile = file;
+      this.form.value.profilePhoto = fileUrl;
```

Comment:
This is not working, it should be something like:

```suggestion
      this.form.patchValue({ profilePhoto: fileUrl });
```

Also, the background image is not being updated.